docs/README.md

```diff
@@ -0,0 +1,23 @@
+## Generating documentation with Sphinx
+
+The best tutorial for generating documentation can be found [here](http://gisellezeno.com/tutorials/sphinx-for-python-documentation.html)
+
+Files in the source directory are generally static and should not need to be updated.  If another python module is created in `../rosette`, then the source may need to be regenerated using
+
+`sphinx-apidoc -f -o source/ ../rosette/`
```

Comment:
This does something different than `make html`.  Or is this command a pre-req for running `make html`?